.github/workflows/CI.yml

```diff
@@ -16,16 +18,20 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
         architecture: x64
+    - name: Run tests
+      shell: bash
+      run: |
+        python -m venv venv
+        source venv/bin/activate
+        pip install -U pip
+        pip install -r build.requirements.txt
+        maturin develop
+        python -m pytest
     - uses: messense/maturin-action@v1
```

Comment:
To prevent the "double build" thing, could you put this step before running tests then tell pip to install from the already packaged wheel on the filesystem? I assume it's still hanging out on the worker somewhere there?